SS14.Admin/Pages/Logs/Index.cshtml

```diff
@@ -10,37 +10,41 @@
     var nextDisabled = (Model.Items.Count == 0) ? "disabled" : "";
     var nextPageIdx = Model.PageIndex + 1;
     var prevPageIdx = Model.PageIndex == 1 ? null : (int?) Model.PageIndex - 1;
+
+    var darkMode = Request.Cookies["darkMode"] == "true";
+    var darkModeClass = darkMode ? "dark" : "light";
+    var textClass = darkMode ? "text-light" : "text-dark";
```

Comment:
You shouldn't need to change the HTML generated, this should be done by CSS.

Reply:
I think I did it like you wanted?